openrr-gui/src/joint_position_sender.rs

```diff
@@ -412,7 +412,7 @@ where
                         // We don't round the input for now. If we do that, we also need to update
                         // the text input that we show to the user.
 
-                        if !duration.is_normal() {
+                        if !duration.is_normal() && duration != 0.0 {
```

Comment:
same as above, i don't want to compare using '!='.
And it should be positive number.

How about

```suggestion
                        if !duration.is_normal() || duration.is_sign_negative() {
```

or something?